Admin.js

```diff
@@ -11,15 +11,16 @@
 
 var Admin = {
 
-    collectionCounters: [],
+    "collectionCounters": [],
 
     /**
      * This function must be called when an ajax call is done, to ensure
      * the retrieved html is properly setup
      *
      * @param subject
      */
-    shared_setup: function(subject) {
+    "shared_setup" (subject) {
```

Comment:
- eslint: Method 'shared_setup' has too many statements (13). Maximum allowed is 10.